static/app/utils/timeSeries/useFetchEventsTimeSeries.tsx

```diff
@@ -142,11 +142,9 @@ export function useFetchEventsTimeSeries<YAxis extends string, Attribute extends
           topEvents,
           groupBy,
           sort: sort ? encodeSort(sort) : undefined,
-          disableAggregateExtrapolation: defined(extrapolate)
-            ? extrapolate
-              ? '0'
-              : '1'
-            : undefined,
+          // Careful! The value of `disableAggregateExtrapolation` does not matter. Any value will turn extrapolation off immediately. Pass `undefined` if extrapolation is on, otherwise pass `"1"`
+          disableAggregateExtrapolation:
```

Comment:
Hmm, wouldn't it be better to just address this on the backend? I think it's fairly standard to do 1 or 0

Reply:
Talked to Tony on Slack, I'll update the backend for this 👍🏻